students/Mintae_K/lesson03/Slicing_lab_exercise.py

```diff
@@ -0,0 +1,31 @@
+def exchange_f_l(x):
+    #print ("with the first and last items exchanged:",s)
+    s = x[-1:] + x[1:-1] + x[0:1]
+    return s
+#
+def every_other(x):
+    #print ("with every other item removed:",s)
+    s = x[::2]
+    return s
+    #next
+def first4last4(x):
+    #print ("first 4, last 4 removed, and every other item in between:",s)
+    s = x[4:-4:2]
+    return s
+#next
+def reverse_sort(x):
+    #print ("with the elements reversed (just with slicing):",s)
+    s = x[::-1]
+    return s
+    #next
+def mid3last3first3(x):
+    third = int(len(x)/3)
+    s = x[third:-third] + x[-third:] + x[:third]
+    return s
```

Comment:
couple small comments:
no need for separate return, instead of:
```
s = x[::-1]
return s
```
return
`return x[::-1]`

and make sure to have two separate blank lines between functions.